src/lib/components/mobile-nav.svelte

```
@@ -1,42 +1,43 @@
-    });
+    let searchModalOpen = $state(false);
 </script>
 
+{#if !isMainPage}
+    <button
+        transition:fly={{ y: -20, duration: 400, easing: cubicOut }}
+        onclick={() => {
+            if (history.length > 1) {
+                history.back();
+            } else {
+                goto('/');
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)